src/temperature/models.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, Integer, ForeignKey, DateTime, Float, String
+from sqlalchemy.orm import relationship
+
+from src.city.models import DBCity
+from src.database import Base
+
+
+class DBTemperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    city_id = Column(Integer, ForeignKey("city.id"))
+    date_time = Column(DateTime)
+    temperature = Column(String(8), nullable=False)
```

Comment:
The `temperature` column is defined as a `String` with a length of 8. If temperatures are stored as numeric values, consider using a `Float` or `Integer` type instead. This will allow for more accurate and efficient storage and querying of temperature data.